src/ResourceManager/Advisor/Commands.Advisor/Cmdlets/GetAzAdvisorRecommendation.cs

```diff
@@ -62,20 +62,43 @@ public class GetAzAdvisorRecommendation : ResourceAdvisorBaseCmdlet
         public string ResourceGroupName { get; set; }
 
         /// <summary>
-        /// Gets or sets the Refresh.
+        /// Gets the list of recommendations associated with the subscription-Id.
         /// </summary>
-        [Parameter(ParameterSetName = "IdParameterSet", Mandatory = false, HelpMessage = "Regenerates the recommendations.")]
-        [Parameter(ParameterSetName = "NameParameterSet", Mandatory = false, HelpMessage = "Regenerates the recommendations.")]
-        public SwitchParameter Refresh { get; set; }
+        /// <returns></returns>
+        internal List<PsAzureAdvisorResourceRecommendationBase> GetRecommendations()
```

Comment:
Please move this out into a common file if you are going to use it in multiple places - that way people know that modifying it will modify multiple cmdlets.